src/Plugin.php

```diff
@@ -141,18 +141,29 @@ private function loadInstalledPaths()
      */
     private function saveInstalledPaths()
     {
-        // By default we delete the installed paths
-        $arguments = ['--config-delete', 'installed_paths'];
+        $configKey = 'installed_paths';
 
-        // This changes in case we do have installed_paths
+        // Check if we found installed_paths to set.
         if (count($this->installedPaths) !== 0) {
-            $arguments = ['--config-set', 'installed_paths', implode(',', $this->installedPaths)];
+            $paths = implode(',', $this->installedPaths);
+            $arguments = ['--config-set', $configKey, $paths];
+            $this->io->write(sprintf('PHP CodeSniffer Config <info>%s</info> <comment>set to</comment> <info>%s</info>', $configKey, $paths));
+        }
+        else {
```

Comment:
Violates [PRS-2: 5.1. if, elseif, else](http://www.php-fig.org/psr/psr-2/#if-elseif-else).
Please correct and place `else` on the same line as the closing bracket of the above if statement.